### PR TITLE
Slide types + Github pages link

### PR DESCRIPTION
**Pull request by Rodak123**
just added a link to GitHub pages to the `README` file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 <img src="./img/title.png">
 
 # Oregon Trail Game
-
+https://codingtrain.github.io/Oregon-Trail/
```
Comment (procgen) [inline README.md]: Maybe use a markdown link:
```md
[Play it](https://codingtrain.github.io/Oregon-Trail/)
```

**Comment by Rodak123** [comment]
Hopefully it allows for an easy way to add new types.

**Comment by shiffman** [comment]
Looking great! I'm happy to merge and conbine with #33. Let me know if it's ready to do so.

Also, we should probably conform our VSCODE settings, a lot of these pull requests are introducing trivial whitespace changes!

**Comment by Rodak123** [comment]
I didn't mean to close it

**Comment by Rodak123** [comment]
> Looking great! I'm happy to merge and conbine with #33. Let me know if it's ready to do so.
> 
> Also, we should probably conform our VSCODE settings, a lot of these pull requests are introducing trivial whitespace changes!

@shiffman 
I'm currently using Sublime Text so I'll switch to VSCODE.

The auto wrapping still needs to be resolved, but you can merge now.

**Comment by Rodak123** [comment]
Here's a preview
Screenshot from 2022-11-23 21-22-42

**Pull request merged**